src/runtime/pay-flow.ts

```diff
@@ -193,8 +193,8 @@ export class PayStartFlow {
       {
         forceRedirect:
           this.deps_.config().windowOpenMode == WindowOpenMode.REDIRECT,
-        // SwG basic and TwG flows do not support native.
-        forceDisableNative: paySwgVersion == '2' || paySwgVersion == '3',
+        // SwG basic does not support native.
+        forceDisableNative: paySwgVersion == '2',
```

Comment:
```suggestion
        forceDisableNative: paySwgVersion === '2',
```

Reply:
Thought about this, but this could break if paySwgVersion is ever passed as a number. Left this out to make sure the chagne is safe.

Reply:
Good call